tests/integration/tests/autocomplete.js

```diff
@@ -71,7 +71,10 @@ test('keyboard navigation', async () => {
   responseHandler.addPreparedResponse(mockAutocomplete, /autocomplete/);
   await page.goto(APP_URL);
   await autocompleteHelper.typeAndWait(TypedSearch);
+  await page.waitForSelector('.autocomplete_suggestions');
+  await page.waitFor(100);
   await autocompleteHelper.pressDown();
+  await page.waitFor(100);
```

Comment:
this seems to be requirer in order to wait for the event to be completed. Let me know if you have a better idea

Reply:
I guess it could be replaced by a selector on the `highlighted` status of the suggest item which is supposed to be selected after a first key down. Maybe we can define a function which press and wait?
But if the test is reliable I would be ok to move on with this implementation and try to refactor it separately.